lib/model/binding.dart

```diff
@@ -152,6 +189,67 @@ class IosDeviceInfo extends BaseDeviceInfo {
   IosDeviceInfo({required this.systemVersion});
 }
 
+/// Like [device_info_plus.MacOsDeviceInfo], but without things we don't use.
+class MacOsDeviceInfo extends BaseDeviceInfo {
+  /// The major release number, such as 10 in version 10.9.3.
+  final int majorVersion;
```

Comment:
Cool, these are good descriptions.

Is there some Apple documentation we can link to for the underlying platform fields they correspond to?